src/libraries/Common/tests/System/Xml/XPath/Common/Utils.cs

```diff
@@ -1,32 +1,50 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Xunit;
 using System;
 using System.Globalization;
 using System.Xml;
 using System.Xml.XPath;
+using Xunit;
 
 namespace XPathTests.Common
 {
     public static partial class Utils
     {
-        public static XPathNavigator CreateNavigatorFromFile(string fileName)
+        private static readonly ICreateNavigator s_xmlDocumentNavigatorCreator = new CreateNavigatorFromXmlDocument();
+        private static readonly ICreateNavigator s_xmlReaderNavigatorCreator = new CreateNavigatorFromXmlReader();
```

Comment:
I can't remember exactly this code anymore but I think there were 3 XPath implementations being tested here:
- XPathDocument
- XmlDocument.CreateNavigator() (aka XmlDocumentXPathNavigator)
- XPathNavigator created from XmlReader

but I'm seeing only 2 being tested here